docs/studio/0004-database-schema.md

```diff
@@ -0,0 +1,25 @@
+# 4. database schema
+
+Date: 2024-06-27
+
+## Status
+
+Accepted
+
+## Context
+
+We need to implement a database schema to store both our pages and folders in our db
+
+## Decision
+
+1. We will combine both page/folders into 1 schema and use a json blob to represent the content of the page instead of separating.
```

Comment:
worth mentioning its a JSONB column in postgres

this also allows us to query within the json using query operators and perform indexing etc